docs/src/components/Pill/Pill.component.tsx

```diff
@@ -0,0 +1,45 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled, { css, FlattenSimpleInterpolation } from 'styled-components';
+// THEME
+import { sizes, fonts } from '../../../../src/theme';
+
+
+// INTERFACES
+interface PillProps {
+  backgroundColor: string;
+  textColor: string;
+  value: number;
+}
+
+
+// STYLES
+export const StyledPill = styled.div`
+  color: ${props => props.textColor };
```

Comment:
We'll wanna set the `box-sizing` of this to `border-box` ([sauce](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing)) to make sure all your padding and margins are predictably rendered without exploding a DOM block.

Reply:
Thanks for this catch 🐟